src/Raven.Studio/typescript/components/pages/resources/manageServer/serverSettings/ServerSettings.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Col, Tooltip } from "reactstrap";
+import { AboutViewHeading } from "components/common/AboutView";
+import { ServerSettingsVirtualTable } from "components/pages/resources/manageServer/serverSettings/ServerSettingsVirtualTable";
+import ButtonWithSpinner from "components/common/ButtonWithSpinner";
+import { Icon } from "components/common/Icon";
+import { ServerSettingsInfoHub } from "components/pages/resources/manageServer/serverSettings/ServerSettingsInfoHub";
+import SizeGetter from "components/common/SizeGetter";
+import { useServices } from "hooks/useServices";
+import { useAsync } from "react-async-hook";
+import { settingsEntry } from "models/database/settings/databaseSettingsModels";
+import { sortBy } from "common/typeUtils";
+import { useAppSelector } from "components/store";
+import { accessManagerSelectors } from "components/common/shell/accessManagerSliceSelectors";
+import FeatureNotAvailable from "components/common/FeatureNotAvailable";
+
+export function ServerSettings() {
+    const { result: serverSettingsData, execute, loading, status } = useFetchServerSettings();
+    const isClusterAdminOrClusterNode = useAppSelector(accessManagerSelectors.isClusterAdminOrClusterNode);
+
+    const handleRefresh = () => execute();
+
+    if (isClusterAdminOrClusterNode) {
+        return (
+            <div id="server-settings-view" className="content-padding h-100">
```

Comment:
I copied from an old view, I suspected it was needed for testing or something else, I also used it in testing as an "anchor"

Reply:
If you actually need it for a test, it should be in `data-testid`. We tend to avoid this approach. Tests should check what the user sees on the screen. Let's check whether, for example, the text/button appears, whether it is disabled, etc.